setup.cfg

```diff
@@ -32,7 +32,7 @@ astropy_header = true
 text_file_format = rst
 xfail_strict = true
 remote_data_strict = true
-addopts = --doctest-rst
+#addopts = --doctest-rst
```

Comment:
Please remove this comment, we started to doctest narrative docs for a couple of modules and eventually would like to see all of them being tested